pkg/autopilot/channels/versions.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 k0s authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package channels
+
+import (
+	"github.com/k0sproject/version"
+)
+
+type DownloadURL struct {
+	Arch         string `yaml:"arch"`
+	OS           string `yaml:"os"`
+	K0S          string `yaml:"k0s"`
+	K0SSha       string `yaml:"k0sSha"`
```

Comment:
Can we be more specific about the hash algo? Either by naming this field, say

```suggestion
	K0sSHA256       string `yaml:"k0sSha256"`
```

... or by having some sort of hash struct `{hash:{type sha256, value: deadbeef}}`.